docs/sources/tempo/setup/operator/monitor.md

```diff
@@ -87,3 +92,21 @@ spec:
       sampling_fraction: "1.0"
       jaeger_agent_endpoint: localhost:6831
 ```
+
+
+## Monitor the operator
+
+The Tempo Operator can expose upgrade and other operational metrics about the operator itself, and can create alerts based on these metrics.
```

Comment:
What do you mean by "can expose upgrade and other operational metrics"?

Reply:
The operator handles Tempo upgrades, and it exposes metrics like "number of successful Tempo upgrades", "number of failed Tempo upgrades". And the operator create alerts, to notify system administrators if for example any Tempo upgrade failed.

The other operational metrics are internals of the operator, for example the duration of a "reconcile loop iteration". That's a operator-specific component which continuously tries to match the expected state as described in the TempoStack custom resource to the actual cluster state. E.g. if you delete some object in the cluster which is managed by the operator, the operator re-creates this object again, to match the expected state of the cluster.

Reply:
This is great info!